userdocs/src/usage/fargate-support.md

```diff
@@ -24,7 +24,7 @@ $ eksctl create cluster --fargate
 [ℹ]  will create 2 separate CloudFormation stacks for cluster itself and the initial nodegroup
 [ℹ]  if you encounter any issues, check CloudFormation console or try 'eksctl utils describe-stacks --region=ap-northeast-1 --cluster=ridiculous-painting-1574859263'
 [ℹ]  CloudWatch logging will not be enabled for cluster "ridiculous-painting-1574859263" in "ap-northeast-1"
-[ℹ]  you can enable it with 'eksctl utils update-cluster-logging --region=ap-northeast-1 --cluster=ridiculous-painting-1574859263'
+[ℹ]  you can enable it with 'eksctl utils update-cluster-logging --enable-types=all --region=ap-northeast-1 --cluster=ridiculous-painting-1574859263'
```

Comment:
this should probably also show the actual message, also below

Reply:
right, done...

Reply:
still missing one..

Reply:
after grepping through all the files, I have not replaced all relevant messages accordingly.